src/components/Navbar/LanguageSelector.module.scss

```diff
@@ -1,8 +1,8 @@
 @use "src/styles/utility";
 @use "src/styles/breakpoints";
 
-$select-width: calc(6.3 * var(--spacing-small));
```

Comment:
awesome,

in PR #800 , we already make the `bahasa melayu` shorter. I think we can reduce the `select-width` here. Maybe 6.5 spacing medium is a good width? What do you think @iyunusov ?

Reply:
@muhajirdev We can reduce it to any size we want. I have made a little bit of research, seems the longest language name to exist is less than or around 20 characters, so with the current setting we are more on the safer side in case in future more languages will be added to the list...  On the other end, Navbar has only few items and there's plenty of empty space after selectbox, we could change `select-width` to `8` instead of `10`, but again any name longer than ex. Portugues which currently is 9 character name, would not appear fully.